generator.rb

```diff
@@ -0,0 +1,71 @@
+# run_generator prompts user for 2 words and prints a portmanteau of the words
+# to the screen
+def run_generator
+  # prompt user for 2 words to combine
+  puts "What is the first word to combine?"
+  word1 = get_word
+  puts "What is the second word to combine?"
+  word2 = get_word
+
+  # WAVE 1 CODE
+  # puts "Word 1: #{word1}"
+  # puts "Word 2: #{word2}"
+  # test_vowel = "i"
+  # puts "Is #{test_vowel} a vowel?"
+  # puts is_vowel?(test_vowel)
+  # test_consonant = "g"
+  # puts "Is #{test_consonant} a vowel?"
+  # puts is_vowel?(test_consonant)
+  # puts "in the run_generator method"
```

Comment:
I would delete this dead code! It just muddies the waters when someone goes to look through it.